### ISSUE TITLE
NITF pipeline tests fail

### ISSUE THREAD

**Issue by chambbj** [PDAL/PDAL#1169]
```
Bradleys-MBP:build chambbj$ bin/pcpipeline_test --gtest_filter=*Nitf*
Note: Google Test filter = *Nitf*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from pipelineNitfTest
[ RUN      ] pipelineNitfTest.chipper
[       OK ] pipelineNitfTest.chipper (31 ms)
[ RUN      ] pipelineNitfTest.conversion
[       OK ] pipelineNitfTest.conversion (31 ms)
[ RUN      ] pipelineNitfTest.las2nitf
PDAL: Can't convert option as requested.

/Users/chambbj/loki/pdal/repo/test/unit/apps/pcpipelineTest.cpp:59: Failure
Value of: stat
  Actual: 1
Expected: 0

[  FAILED  ] pipelineNitfTest.las2nitf (30 ms)
[ RUN      ] pipelineNitfTest.write_laz
PDAL: Invalid option name 'FSCLASS'.  Options must consist of only lowercase letters, numbers and '_'.

Assertion failed: (Option::nameValid(option.getName(), true)), function add, file /Users/chambbj/loki/pdal/repo/src/Options.cpp, line 97.
[       OK ] pipelineNitfTest.write_laz (182 ms)
[ RUN      ] pipelineNitfTest.write_options
PDAL: The given TRE is already managed by the library. Try cloning it first.

/Users/chambbj/loki/pdal/repo/test/unit/apps/pcpipelineTest.cpp:59: Failure
Value of: stat
  Actual: 1
Expected: 0

[  FAILED  ] pipelineNitfTest.write_options (46 ms)
[----------] 5 tests from pipelineNitfTest (320 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (320 ms total)
[  PASSED  ] 3 tests.
[  FAILED  ] 2 tests, listed below:
[  FAILED  ] pipelineNitfTest.las2nitf
[  FAILED  ] pipelineNitfTest.write_options

 2 FAILED TESTS
  YOU HAVE 1 DISABLED TEST
```

I'll push a branch with updated XMLs to demonstrate shortly.

**Comment by chambbj** [comment]
https://github.com/PDAL/PDAL/tree/issue/1169-nitf-tests

**Comment by abellgithub** [comment]
Fixed with e9c73a7a89e3fea62853986133a637ba201de583
